### PR TITLE
Migrate pull off linux-xenial-py3_7-clang7-onnx

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

**Comment by justinchuby** [comment]
Could you help update

https://github.com/pytorch/pytorch/blob/543919cfc8d7014dfa365b55c459fa2b3f746891/.github/merge_rules.json#L38-L39

too

**Comment by kit1980** [comment]
> Could you help update
> 
> https://github.com/pytorch/pytorch/blob/543919cfc8d7014dfa365b55c459fa2b3f746891/.github/merge_rules.json#L38-L39
> 
> too

I'll update in a separate PR, to let this PR merge without force.

**Comment by kit1980** [comment]
> I think you can update whenever, since you'll pass the superuser rule.
> For the onnx one, should we be moving that to "pull" in general as the rest of the rules are pull now?

Yes, the previous comment is obsolete now as the merge rules are "pull" now.

**Review by malfet — APPROVED** [review]

File: .circleci/docker/build.sh [hunk 1]
```diff
@@ -176,6 +176,13 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
```
[the remark this answers is not in the document]
Comment (kit1980) [reply .circleci/docker/build.sh:178]: I'll do clean-up in a separate PR after double checking it's no longer needed anywhere.

**Pull request closed without merging**